models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md

```diff
@@ -54,7 +54,7 @@ Expected color order: BGR.
    where:
 
     - B - batch size
-    - C - vector of 3 values in format [H,W,S], where H - image height, W - image width, S - image scale factor (usually 1)
+    - C - vector of 3 values in format [H,W,S], where H - image height, W - imahe width, S - image scale factor (usually 1)
```

Comment:
commit seems not related to this PR

Reply:
Rebased to the current develop, could you please recheck this?